2-mandatory/1-writers.js

```diff
@@ -72,6 +77,10 @@ Exercise 2:
 
 function logDeadWritersInTheirForties() {
   // write your code here
+  writers.forEach(({ age, lastName, firstName, alive }) => {
+    if (age > 39 && age < 50 && !alive)
```

Comment:
logical comparison starts left to right so you may start comparing alive first to save possible operations